packages/next/pages/_document.js

```diff
@@ -201,7 +203,7 @@ export class Head extends Component {
               name="viewport"
               content="width=device-width,minimum-scale=1,initial-scale=1"
             />
-            <link rel="canonical" href={page} />
+            <link rel="canonical" href={page.split('.amp')[0]} />
```

Comment:
What's this result in? Seems weird

Reply:
It takes `/dog.amp` and turns it into `/dog`

Reply:
`/dog/lab.amp` => ?

Reply:
Hmm I guess it just seems strange to have a relative href in a canoncial tag

Reply:
`/dog/lab.amp` will be => `/dog/lab`

Reply:
Didn't realize `page` was the path